src/makeRobot.js

```diff
@@ -39,6 +39,84 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+
+    goForward(forward) {
+      if (forward < 0) {
+        return this;
+      } else if (typeof forward === 'number') {
+        this.coords.y += forward;
+      } else {
+        this.coords.y += 1;
+      }
```

Comment:
What does `forward` mean? My assumption it better call `step` and give default value. In that case you no need do a lot of checks. Just check if value ,more than 0. 

Fix all places
```suggestion
    goForward(steps = 1) {
      if (steps> 0) {
        this.coords.y += steps ;
      } 
```

Reply:
Thank you for cool recommendation!